package.json

```diff
@@ -5,7 +5,7 @@
     "main": "index.js",
     "scripts": {
         "pretest": "node node_modules/rimraf/rimraf coverage/* reports/*",
```

Comment:
It turns out that this rimraf line silently does nothing. Execute rimraf/bin.js instead

Reply:
Good catch.  Fixed.